dist/js/admin.js

```diff
@@ -43,6 +44,13 @@ if ( document.querySelectorAll( '.remove-payment-method' ).length !== 0 ) {
 		}
 	);
 }
+if ( document.querySelectorAll( '.routing' ).length !== 0 ) {
+	document.querySelectorAll( '.routing' ).forEach(
```

Comment:
No need of the first conditional I think, if the result is empty, `forEach` won't iterate. Alternatively I would assign the result of `document.querySelectorAll` to a variable to avoid querying the DOM multiple times.

Reply:
Yes... great. Collections checks removed, DOM queries assigned to variables and dropdowns implemented for newly added payment methods.